src/modules/NoiseReduction/Module.js

```diff
@@ -0,0 +1,43 @@
+module.exports = function NoiseReduction(options, UI){
+  var defaults = require('./../../util/getDefaults.js')(require('./info.json'));
+  var output;
+  
+  function draw(input,callback,progressObj){
+  
+    progressObj.stop(true);
+    progressObj.overrideFlag = true;
+  
+    var step = this;
+    options.method = options.method || defaults.method;
+
+    function changePixel(r, g, b, a){
```

Comment:
@aashna27 Please remove the changePixel function. With the changes I made to pixelManipulation, you should just be able to omit passing the changePixel function rather than passing an identity function!

Reply:
Thanks a lot @tech4GT I ll just make the changes. :smile_cat: